src/shx.js

```diff
@@ -18,7 +18,7 @@ const CMD_BLACKLIST = [
   'exec',
 ];
 
-const shx = (argv) => {
+export const shx = (argv) => {
```

Comment:
In order to more easily spy/stub default exports, I moved us to named exports.  This gives us an import object whose properties we can easily stub. Reference https://github.com/sinonjs/sinon/issues/664.